library/insights_config.py

```diff
@@ -51,6 +51,7 @@
       password: "rhn_password" or "{{ redhat_portal_password }}"
       auto_config: False or "{{ auto_config }}"
       authmethod: BASIC or "{{ authmethod }}"
+      proxy: "{{ insights_proxy }}"
```

Comment:
Could you add a short description in the Documentation section above? Other than that I think we're good!